src/scripts/version_history.sh

```diff
@@ -23,9 +23,10 @@ fi
 # Some apps may not have set the correct k8s label with a valid app version containing a sha1
 # Check $previous_commit sha1 is valid
 if git rev-parse --quiet --verify "${previous_commit}" &>/dev/null; then
+  # shellcheck disable=SC2086
   PAGER="cat" git log --oneline --no-decorate \
     --pretty=format:'%h %s (%cr)' --committer='noreply@github.com' --grep='#' \
-    "${previous_commit}..${current_commit}" \
+    "${previous_commit}..${current_commit}" $CHANGELOG_GIT_PATHS \
```

Comment:
default is `""` which will show everything.  Had to disable the globbing shell check since quoting the variable then passed `""` into the `git log` command which then didn't find anything